libdevcrypto/Blake2.cpp

```diff
@@ -0,0 +1,160 @@
+/// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+#include "Blake2.h"
+
+#include <libdevcore/Exceptions.h>
+
+// The Blake 2 F compression function implemenation is based on the reference implementation,
```

Comment:
@chfast do we need any kind of license note here?

Reply:
It is not required because the reference implementation is in public domain, but I think it is nice to leave it here.